### DISCUSSION TITLE
Raw Elasticsearch vs SO

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#3412] [Unsupported Versions] [answered]
Elasticsearch/Kibana comes with a lot of pre-built search and dashboards to visualise data from sources such as firewalls (ASA/Fortigate/PA), zeek, suricata etc. and the ingest pipelines to understand these logs. Also flow data.
SO comes with some REALLY great alert visualisations and integration with other tools like Hive.

What would be FANTASTIC is if we could leverage the built-in Elastic functionality to visualise and search some of the datasources, AND get the SO integrations.  

The SO install of Elastic seems to be "minimalistic", just providing the functionality required for the SO visualisations etc.
Has anyone come up with a way to layer the SO on top of a complete Elastic install to get the best of both worlds?

**Comment by weslambert** [comment, marked as answer]
Thanks for the feedback.  You should still be able to use Elastic Features (ex. parsing from various FB modules, etc) and build your own visualizations for whatever you like.  There is nothing stopping you from doing this, currently.
